### PR TITLE
Ada 26 master

### PR DESCRIPTION
**Pull request by cachemeifyoucan0**
to test:

- create a meeting in the database that has an event code, since this ticket assumes that exists (can do this by logging into exec and clicking on meetings, and creating one or selecting one that's there)
- go to "https://dev-ada.herokuapp.com/attendance"
- fill out the form with the code created and submit
- check database to see if successful created new student if needed and to meeting_student table

### CODE REVIEW

**Review by jgeorge37 — CHANGES_REQUESTED** [review]
When I submit attendance for the same person for two different meetings, but I change the year of school the person is in, the year does not get updated in the database - i.e. over multiple academic years someone cannot change their year in school.

Also, if I fill out the form for a student for one meeting, their attendance is recorded, but when I fill it out for the same student for a different meeting, their attendance is not recorded. I also got a 500 internal server error in this case.

**Comment by cachemeifyoucan0** [comment]
> When I submit attendance for the same person for two different meetings, but I change the year of school the person is in, the year does not get updated in the database - i.e. over multiple academic years someone cannot change their year in school.
> 
sorry I didn't realize this had to cover over multiple years! good catch
> Also, if I fill out the form for a student for one meeting, their attendance is recorded, but when I fill it out for the same student for a different meeting, their attendance is not recorded. I also got a 500 internal server error in this case.
hm that's odd thought I def tested this I'll double check 

also - are the accounts on dev back? I just wasn't to reset my password

**Comment by jgeorge37** [comment]
ok i just realized i accidentally remade all of the accounts with the .420 student name dot nums that angela put in there instead of the real name dot nums oops

**Comment by cachemeifyoucan0** [comment]
> ok i just realized i accidentally remade all of the accounts with the .420 student name dot nums that angela put in there instead of the real name dot nums oops

lmao kk tanks

ok I fixed the year update, but I can't duplicate the double post in my local, and still my forgot password for mason.906 didn't go thru in dev

File: acmw-db-app/pages/api/attendance/[pid].js [hunk 1]
```diff
@@ -26,7 +26,16 @@ async function record (event_code, f_name, l_name_dot_num, year_level) {
        curr_student_id = await createStudent(f_name, l_name, l_name_dot_num, "", year_level, "");
    }
 
-    console.log(curr_student_id);
+    //school_level
+    // need to update student year level if it has changed
+    let current_student_yr = await pgQuery(`SELECT school_level FROM student WHERE name_dot_num = '${l_name_dot_num.toLowerCase()}'`);
```
Comment (jgeorge37) [inline acmw-db-app/pages/api/attendance/[pid].js:31]: I wouldn't bother checking if it is different - I'd rather have one query in every case than possibly 2 queries in some cases. If it hasn't changed no damage is done with update.

**Review by jgeorge37 — CHANGES_REQUESTED** [review]

**Review by jgeorge37 — APPROVED** [review]

**Pull request merged**